themes/edx.org/lms/templates/footer.html

```diff
@@ -18,83 +19,269 @@
     dir=${bidi}
   % endif
 >
-    <h2 class="sr footer-about-title">${_("About edX")}</h2>
-    <div class="footer-content-wrapper">
-      <div class="footer-logo">
-          <a href="${marketing_link('ROOT')}">
-            <img alt="${_('edX Home Page')}" src="${footer['logo_image']}">
-          </a>
-      </div>
+    <style scoped>
```

Comment:
You should not be including Bootstrap CSS in the footer, as the page might not be using Bootstrap. You should rely on the page having the correct styling available.

Reply:
This is temporary and will be removed

Reply:
👍